common-files/.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contribution guide
```

Comment:
Do we need to have this at all? We could just let the new plugin default to using https://github.com/jenkinsci/.github/blob/master/CONTRIBUTING.md right? (Which should be amended to give some of this information.) Are you copying this text from some standardized place? I would really prefer not to have to maintain the rather detailed instructions here. CC @oleg-nenashev

Reply:
Tested and that works better, will move it there / adjust as needed.

it came from https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/CONTRIBUTING.md I think